src/orderbook.ts

```diff
@@ -77,3 +93,16 @@ function addOffer(offer: Offer, existingOffers: Map<number, Offer>) {
 function sortOffersAscending(left: Offer, right: Offer) {
   return left.price.toNumber() - right.price.toNumber();
 }
+
+function invertPricePoints(prices: Map<number, Offer>) {
+  return new Map(
+    Array.from(prices.entries()).map(([_, offer]) => {
```

Comment:
ubernit: It's so common this patter of converting an iterator into a map, that`Array.from` has already a second argument for this, so no need to chain a map there.

Also, if you want to discard a value of an array, you can just omit it `[, offer]`